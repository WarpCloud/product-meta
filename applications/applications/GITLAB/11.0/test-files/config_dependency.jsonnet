{
  application_type: "GITLAB",
  application_name: "gitlab1",
  application_version: "11.0",
  user_config: {},
}
