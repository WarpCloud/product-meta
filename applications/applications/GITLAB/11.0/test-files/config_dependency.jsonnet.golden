{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "gitlab1",
            "application_type": "GITLAB"
         },
         "configs": {
            "gitlab_cpu_limit": 1,
            "gitlab_cpu_request": 1,
            "gitlab_memory_limit": 1,
            "gitlab_memory_request": 1,
            "postgresql_cpu_limit": 1,
            "postgresql_cpu_request": 1,
            "postgresql_memory_limit": 1,
            "postgresql_memory_request": 1,
            "redis_cpu_limit": 1,
            "redis_cpu_request": 1,
            "redis_memory_limit": 1,
            "redis_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "gitlab",
         "name": "gitlab1-gitlab",
         "version": "11.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
